scripts/generate-deploy-config.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/sh
+
+set -e
+
+config_dir=tasks/deploy/config
+
+factories=()
+coordinators=()
+instances=()
+
+function update_indexes(){
```

Comment:
Let's make it a habit to add at least a simple comment to every code-block. It makes code nicer to read IMO